pkg/api/index.go

```diff
@@ -327,7 +327,7 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 		navTree = append(navTree, &dtos.NavLink{
 			Id:           "live",
 			Text:         "Live",
-			SubTitle:     "Event Streaming",
+			SubTitle:     "Event streaming",
```

Comment:
@lean257 @FZambia (I think you own this feature?) While I was checking this file for translations, I noticed this nav item didn't follow the proper sentence case convention, so I fixed it up. Just checking in case there's any objections?